tests/openvino/test_exporters_cli.py

```diff
@@ -107,6 +107,8 @@ class OVCLIExportTestCase(unittest.TestCase):
         ("text-generation-with-past", "opt125m", "int4 --sym --group-size 128", {"int8": 4, "int4": 72}),
         ("text-generation-with-past", "opt125m", "int4 --group-size 64", {"int8": 4, "int4": 144}),
         ("text-generation-with-past", "opt125m", "mxfp4", {"int8": 4, "f4e2m1": 72, "f8e8m0": 72}),
+        ("text-generation-with-past", "opt125m", "nf4", {"int8": 4, "nf4": 72}),
+        ("text-generation-with-past", "opt125m", "nf4  --awq --dataset wikitext2", {"int8": 4, "nf4": 72}),
```

Comment:
The tests are pretty slow already. Can we just keep data-free test for "nf4"? It should be enough at this point.